### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by aizjicod**
on this project I worked on the cards section as well on the popup of the same, hence I did the following: 

- I first created a template for the popup
- I deleted the cards in order to create them dynamically with js
- later on, then I proceeded to work with js files and data to create the cards on it.
- then I proceeded to create the popup for each of them with js.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -80,104 +80,10 @@ <h2>Multi-Post Stories</h2>
                     <a><div class="tag-btn">Ruby</div></a>
                 </div>
                 <div class="works__action-btn">
-                    <button type="button" class="works-btn">See Project</button>
+                    <button type="button" class="works-btn project-btn">See Project</button>
                 </div>
```
Comment (KDlamini) [inline index.html:84]: Kindly add a popup for the "Multi-Post Stories" project as well. Currently, nothing happens when I click on the "See Project" button.

image

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi team @aizjicod, @renercrows 🙋‍♀️,  

Great job so far! 💯 
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 
✔️  Popup is responsive
✔️  Popup follows FIGMA design
✔️  JavaScript code is implemented following the best practices
✔️  JavaScript and HTML&CSS linters is set correctly
✔️  No linter errors

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js [hunk 2]
```diff
@@ -1,18 +1,208 @@
...
+  },
+  {
+    name: 'Profesional Art Printing Data 4',
+    info: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    tec: ['bootstrap', 'css', 'html', 'ruby'],
+    img: 'img/card-bg.png',
+    seelive: 'link-to-other-ptoject',
+    linkCode: 'link-to-the-repository',
+  },
+  {
```
Comment (koachcode) [inline script.js:63]: - [Optional] Please try making your popup content look like that of the Figma design, in spacing, and width. For instance, the width of the project image in your popup is small and that of the Figma design is large, with spacing between the title and technology use, and button shape. Please try to follow the Figma design on this.

##### Your design 
Screenshot from 2022-06-15 18-45-50

##### Figma design
Screenshot from 2022-06-15 18-46-23

**Review by koachcode — APPROVED** [review]
Hi @aizjicod , 

# STATUS: APPROVED 🟢

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**